L-Carr/Platoon-Console
# 65 watch code platoon videos from dashboard frontendbackend youtube api implementationstretch goal

## What type of PR is this? (check all applicable)

- [ ] Other (please specify here: <type here>)
  
  
## Description of what this Pull Request does
Updated .scss to include styling for modals in the VideoGallery.jsx file
Refactored code & added search query feature for YouTube
Updated router.jsx with proper path "...videos/"
react-modal installed 

Still require additional features:
*Admin has ability to to add/set Playlist & Cohort (Flexibility)
*Admin Assoc. 1 or more playlists w/ a cohort.
*Option to filter by Cohort/Topic
*Order By Option: (Date/Title)
*Search query is awesome
*Add tags below search bar (Nice 2 have. Not needed)
*Favorites (Students can Star videos to save in a list)


## Related Tickets & Documents
- Closes #65 
  
## QA Instructions, Screenshots, Recordings
*Pull and review to see what it is like. Let me know about any additional ideas, etc.
  
## Added/updated tests?
- [X] Not Yet, and here is why: N/A (When completed, and time permitting, will implement)
- [ ] I need help with tests
- [ ] No, and here is why: __explain here__
